src/main/java/billgates/entities/UserFactory.java

```diff
@@ -0,0 +1,73 @@
+package billgates.entities;
+
+public class UserFactory {
+    //use getUser method to get object of type that implements Iuser
+    private int temporaryID;
+    private String temporaryName;
+    public String temporaryPW = "temporaryPW";
+    private int temporaryBillID =0;
```

Comment:
Style: `private int temporaryBillID =0;` -> `private int temporaryBillID = 0;` (a space after the equal operator)